torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -389,7 +391,7 @@ def runtime_wrapper(args: List[Any]):
                 else:
                     t._dynamo_weak_dynamic_indices = o.dynamic_dims.copy()
         if runtime_metadata.grad_enabled_mutation is not None:
-            torch.set_grad_enabled(runtime_metadata.grad_enabled_mutation)
+            torch._C._set_grad_enabled(runtime_metadata.grad_enabled_mutation)
```

Comment:
is there so much overhead calling `torch.set_grad_enabled` vs calling the C API directly?

Reply:
```python
In [1]: import torch
   ...: %timeit torch.set_grad_enabled(False)
   ...: %timeit torch._C._set_grad_enabled(False)
536 ns ± 3.02 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
217 ns ± 1.28 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```

It's 2.5 x slower, mostly because `torch.set_grad_enabled` is a context manager object that pretends to be a normal function.